src/scenes/Event/index.js

```diff
@@ -76,11 +77,18 @@ export default class EventPage extends Component {
   }
 
   getEventDesc = () => {
-    const { event, event: { status } } = this.props.store.eventPage;
+    const {
+      store: {
+        eventPage: { event, event: { status, centralizedOracle } },
+        wallet: { currentAddress },
+      },
+    } = this.props;
     switch (status) {
       case EVENT_STATUS.CREATED: return 'creating';
-      case EVENT_STATUS.BETTING: return moment.unix().isBefore(moment.unix(event.betStartTime)) ? 'predictionComingSoon' : 'predictionInProgress';
-      case EVENT_STATUS.ORACLE_RESULT_SETTING: return moment.unix().isBefore(moment.unix(event.resultSetEndTime)) ? 'resultSettingComingSoon' : 'resultSettingInProgress';
+      case EVENT_STATUS.BETTING: return moment().isBefore(moment.unix(event.betStartTime)) ? 'predictionComingSoon' : 'predictionInProgress';
```

Comment:
seems to be exact same logic as the EventCard. good use case for helper func? or maybe ref same logic in the store?